src/AzureClient/Visualization/TargetStatusEncoders.cs

```diff
@@ -21,7 +21,7 @@ internal static class TargetStatusExtensions
             {
                 ["id"] = target.Id,
                 ["current_availability"] = target.CurrentAvailability,
```

Comment:
Does this one require a default value as done in line 24 below?

Reply:
Yes, that's a good catch, we should be consistent here.